app/ai/voice/agents/breeze_buddy/services/telephony/utils.py

```
@@ -10,16 +12,18 @@
+    aiohttp_session,
+    telephony_config: Optional[TelephonyConfig] = None,
 ) -> VoiceCallProvider:
     if provider_name == CallProvider.EXOTEL:
-        return ExotelProvider(aiohttp_session)
+        return ExotelProvider(aiohttp_session, telephony_config)
     if provider_name == CallProvider.TWILIO:
-        return TwilioProvider(aiohttp_session)
+        return TwilioProvider(aiohttp_session, telephony_config)
     if provider_name == CallProvider.PLIVO:
         return PlivoProvider(aiohttp_session)
     raise ValueError(f"Unsupported voice provider: {provider_name}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)